modules/deploy/pages/deployment-option/cloud/configure-private-service-connect-in-cloud-ui.adoc

```diff
@@ -0,0 +1,97 @@
+
+Consider using the endpoint services if you have multiple VPCs and could benefit from a more simplified approach to network management:
+
+* Private Service Connect allows overlapping xref:./cidr-ranges.adoc[CIDR ranges] in VPC networks.
+* Private Service Connect does not limit the number of connections using the service.
+* You control which GCP projects are allowed to connect to the service.
+
+== Requirements
+
+* Use https://cloud.google.com/sdk/docs/install[gcloud^] to create the consumer-side resources, such as a client VPC and forwarding rule, or modify existing resources to use the Private Service Connect service attachment created for your cluster.
+* The client VPC must be in the same region as your Redpanda cluster.
+
+== Enable endpoint service for existing clusters
+
+. In the Redpanda Cloud UI, open your https://cloud.redpanda.com/clusters[cluster^], and click **Cluster settings**.
+. Under Private Service Connect, click **Enable**. 
+. For xref:deploy:deployment-option/cloud/vpc-byo-gcp.adoc[BYOC clusters with customer-managed VPC], you need a NAT subnet with the *Purpose* set to `PRIVATE_SERVICE_CONNECT`. You can create the subnet using the `gcloud` command-line interface (CLI):
++
+[,bash]
+----
+gcloud compute networks subnets create <subnet-name> \
+    --project=<project> \
+    --network=<network-name> \
+    --region=<region> \
+    --range=<subnet-range> \
+    --purpose=PRIVATE_SERVICE_CONNECT
+----
++
+Provide your values for the following placeholders:
++
+- `<subnet-name>`: The name of the NAT subnet.
+- `<project>`: The **host** GCP project ID.
+- `<network-name>`: The name of the VPC being used for your Redpanda Cloud cluster.
+- `<region>`: The region of the Redpanda Cloud cluster.
+- `<subnet-range>`: The CIDR range of the subnet. The mask should be at least `/29`. Each Private Service Connect connection takes up one IP address from the NAT subnet, so the CIDR must be able to accommodate all projects from which connections to the service attachment will be issued.
++
+See the official GCP documentation for https://cloud.google.com/vpc/docs/configure-private-service-connect-producer#add-subnet-psc[creating a subnet for Private Service Connect^].
+. For the accepted consumers list, you need the GCP project IDs from which incoming connections will be accepted.
+. It may take several minutes for your cluster to update. When the update is complete, the Private Service Connect status in **Cluster settings** changes from **In progress** to **Enabled**.
+
+NOTE: For help with issues with enabling Private Service Connect, contact https://support.redpanda.com/hc/en-us/requests/new[Redpanda support^].
+
+=== Deploy consumer-side resources
```

Comment:
@0x5d I put together this section based on the [Figma design](https://www.figma.com/design/prK7Y5IYW8jbMNYwXw02Hw/PrivateLink?node-id=117-2786&t=t6H5kLnnBivuShFR-0) as well as these original set of docs: https://github.com/redpanda-data/cloudv2/pull/12933/files#diff-b4884caa55c76586fff9375cb88a5e644b0f73f15bf9aa9b5f5d84b4e0fc1292R337 could you confirm if these steps are correct, and if they should also be added to the [API guide](https://deploy-preview-516--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/gcp-private-service-connect/)? In that case, we should probably single source this section to share it across both guides.